roles/foreman_installer_devel_scenario/templates/katello-devel-answers.yaml

```diff
@@ -6,11 +6,8 @@ certs:
 katello_devel:
   deployment_dir: /home/vagrant
   user: vagrant
-  db_type: postgres
-  rvm: true
 foreman_proxy_content:
-  pulp_master: true
-  qpid_router_broker_addr: localhost
+  proxy_pulp_yum_to_pulpcore: true
 foreman_proxy:
   custom_repo: true
```

Comment:
If you want to clean up: this parameter is gone from the module. tftp is also off by default nowadays, same for templates.